models/comments.model.js

```diff
@@ -29,3 +29,34 @@ exports.fetchCommentsByReviewId = (reviewId) => {
     })
 }
 
+
+exports.createReviewComment = (reviewId, postBody) =>  {
+    const { body, username } = postBody; 
+    if (!Number(reviewId)) {
+        return Promise.reject({ status: 400, msg: "Invalid review id"})
+    }
+    const postLength = Object.keys(postBody).length; 
+
+    if (postBody.hasOwnProperty('username' && 'body') === false || postLength !== 2) {
+        return Promise.reject({ status: 400, msg: "Invalid input"})
+    }
+
+    
+    const queryString =
+        `INSERT INTO comments
+         (body, author,review_id) 
+         VALUES ($1, $2, $3) 
+         RETURNING *;`
+    
+
+    return fetchReview(reviewId).then((result) => { 
+        if (!result) {
```

Comment:
I think you could get rid of this if statement - if no review is found by `fetchReview`, `fetchReview`'s own error handling should throw and deal with the error